tests/robot/suites/crud/tap_crud.robot

```diff
@@ -109,3 +94,22 @@ TestSetup
 TestTeardown
     Make Datastore Snapshots    ${TEST_NAME}_test_teardown
 
+Check TAP interface State
```

Comment:
also change the input instead of specific values  for checking use array of @{desired_states}  ( more flexible for reusing )